modules/ROOT/pages/7.6.0-release-notes.adoc

```diff
@@ -151,12 +151,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[bug-fixes]]
 == Bug fixes
 
-{productname} {release-version} also includes the following bug fix<es>:
+{productname} {release-version} also includes the following bug fixes:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Numbered table context menu not properly applying changes
+// #TINY-11383
 
-// CCFR here.
+Previously, an issue was identified where changes to the row type in numbered tables were not being applied. This occurred because the action of modifying the type of a CEF (Content Editable False) cell was being blocked.
+
+In TinyMCE 7.6.0, this issue has been resolved by removing the restriction on changing the type of CEF cells, resulting in changes to the row type now being properly applied.
```

Comment:
@FarzadHayat Nice suggestion, one edit if possible: Can you update your copy suggestion to refer to `+contentEditable="false"+` rather than `Content Editable False (CEF)` for consistency with the rest of our docs 🥺